index.dd

```diff
@@ -96,6 +96,21 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE
+----
+// std.process simplifies invoking external programs
+void main()
+{
+    import std.stdio, std.process;
+
+    auto result = ["echo", "hello", "world"].execute;
+    result.output.write; // hello world
+    writefln("All good?: %d, result.status); // 0
```

Comment:
Unclosed string literal (`%d,` should be `%d",`)

Subjective but I'd put an enforce here instead of writeln :)